lib/xcode.js

```diff
@@ -144,6 +144,31 @@ const simulatorDevicePairCompatibility = {
 			'5.x': true,        // watchOS 5.x
 			'6.x': true         // watchOS 6.x
 		}
+	},
+	'12.x': {              // Xcode 12.x
+		'10.x': {          // iOS 10.x
+			'2.2': true,   // watchOS 2.2
+			'3.x': true    // watchOS 3.x
+		},
+		'11.x': {		   // iOS 11.x
+			'>=3.2': true, // watchOS 3.2
+			'4.x': true    // watchOS 4.x
+		},
+		'12.x': {		   // iOS 12.x
+			'4.x': true,    // watchOS 4.x
+			'5.x': true,    // watchOS 5.x
+			'6.x': true    // watchOS 6.x
+		},
+		'13.x': {		   // iOS 13.x
+			'4.x': true,    // watchOS 4.x
+			'5.x': true,    // watchOS 5.x
+			'6.x': true    // watchOS 6.x
+		},
+		'14.x': {		   // iOS 14.x
```

Comment:
Xcode 12 allows you to pair watchOS 4 simulators with iOS 14 simulators, so this needs:
```
'4.x': true         // watchOS 4.x
```